chapters/ch25.md

```diff
@@ -1,31 +1,32 @@
 # Techniques for reducing avoidable bias
 ->
-
+# Các kĩ thuật để giảm độ chệch tránh được
 
 If your learning algorithm suffers from high avoidable bias, you might try the following techniques:
 ->
-
+Nếu thuật toán học của bạn gặp vấn đề với độ chệch tránh khỏi lớn, bạn có thể thử những kĩ thuật sau:
 
 * **Increase the model size** ​(such as number of neurons/layers): This technique reduces bias, since it should allow you to fit the training set better. If you find that this increases variance, then use regularization, which will usually eliminate the increase in variance.
 ->
+* **Tăng kích thước mẫu** (ví dụ như số lượng neuron/tầng): Kĩ thuật này làm giảm độ chệch, vì nó cho phép fit tập huấn luyện tốt hơn. Nếu bạn thấy việc này làm tăng phương sai, hãy sử dụng regularization, vốn thường triệt tiêu việc tăng phương sai.
 
 
 * **Modify input features based on insights from error analysis​**: Say your error analysis inspires you to create additional features that help the algorithm eliminate a particular category of errors. (We discuss this further in the next chapter.) These new features could help with both bias and variance. In theory, adding more features could increase the variance; but if you find this to be the case, then use regularization, which will usually eliminate the increase in variance.
 ->
-
+* **Thay đổi các đặc trưng đầu vào dựa trên nhận thức rõ có được từ phân tích lỗi ​**: Giả sử việc phân tích lỗi gợi ý bạn rằng hãy tạo thêm những đặc trung bổ sung nhằm giúp thuật toán có thể loại bỏ một kiểu lỗi đặc thù. (Chúng ta sẽ bàn vấn đề này kĩ hơn ở chương sau.) Những đặc trưng mới này có thể hiệu quả với cả độ chệch và phương sai. Trong lý thuyết, thêm đặc trưng có thể làm tăng phương sai; nhưng nếu bạn thấy đúng là điều đó xảy ra, hãy sử dụng regularization, vốn thường triệt tiêu việc tăng phương sai.
 
 * **Reduce or eliminate regularization​** (L2 regularization, L1 regularization, dropout): This will reduce avoidable bias, but increase variance.
 ->
-
+* **Giảm hoặc triệt tiêu regularization​** (L2 regularization, L1 regularization, dropout): Việc này sẽ làm giảm độ chệch có thể tránh được, nhưng sẽ làm tăng phương sai.
```

Comment:
dùng "triệt tiêu" chắc cũng ok. Nhưng với mình khi nhìn thấy từ triệt tiêu thì mình nghĩ đến công thức kiểu như a-b, hai cái đó triệt tiêu nhau; trong ngữ cảnh này thì "triệt tiêu" = "loại bỏ"?

Reply:
Chữ "loại bỏ" tốt hơn nhiều. 👍 
Khi mình dịch mình cũng đã lăn tăn chữ "triệt tiêu" vì đúng là nó mang sắc thái trong toán có vẻ không tự nhiên lắm ở đây (VD phân số có cùng thừa số a trên tử và mẫu nên a "triệt tiêu") -- mà không nghĩ ra alternative choice đơn giản thế này.